easybuild/easyconfigs/p/Python/Python-3.7.2-GCCcore-8.2.0.eb

```diff
@@ -10,6 +10,7 @@ toolchainopts = {'pic': True}
 
 source_urls = ['http://www.python.org/ftp/%(namelower)s/%(version)s/']
 sources = [SOURCE_TGZ]
+patches = ['Python-3.7.2-fix_faulthandler.patch']
 checksums = ['f09d83c773b9cc72421abba2c317e4e6e05d919f9bcf34468e192b6a6c8e328d']
```

Comment:
Checksum for patch is missing. Use `eb Python-3.7.2-GCCcore-8.2.0.eb --inject-checksums --force` to add it automatically (in the right format)

Reply:
Ah, sorry about that.

That command changed a good deal more than just the checksums!  It looks like it is mostly reordering where the 'modulename' key and value for items in the `exts_list` appear.  Rather than possibly muddy the PR, I extracted the checksums line and added it manually.  Hope that was OK.

Reply:
That sounds good (remember to put in the updated checksum after adding the requested text to the patch)

Reply:
Both changes should be in the commit just pushed.